src/const.ts

```diff
@@ -1,92 +1,94 @@
-export const ACL_REGEX = /^[a-zA-Z]+:([a-zA-Z]+|[*]):(([a-zA-Z]+|[*])|([a-zA-Z]+[*]))$/
+export const ACL_REGEX = /^[a-zA-Z]+:[a-zA-Z]+:(([a-zA-Z]+|[*])|([a-zA-Z]+[*]))$/
 
-const competencyPrefix = "competency:competencies";
-const audiencePrefix = "competency:audience";
-const conditionPrefix = "competency:condition";
-const documentationPrefix = "competency:documentation";
-const behaviorPrefix = "competency:behavior";
-const degreePrefix = "competency:degree";
-const effectivenessPrefix = "competency:effectiveness";
-const userPrefix = "competency:user";
-const apiKeyPrefix = "competency:apiKey";
-const searchPrefix = "competency:search";
-const lifecyclePrefix = "competency:lifecycle";
+const competencyService = "competency";
+
+const competenciesModule = "competencies";
+const audienceModule = "audience";
+const conditionModule = "condition";
+const documentationModule = "documentation";
+const behaviorModule = "behavior";
+const degreeModule = "degree";
+const effectivenessModule = "effectiveness";
+const userModule = "user";
+const apiKeyModule = "apiKey";
+const searchModule = "search";
+const lifecycleModule = "lifecycle";
```

Comment:
Do you think it would be better to use enums here instead of individual const variables?

Reply:
Thanks for the suggestion! Looks a lot cleaner!